src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String MY_CHARACTER = "w";
+    private static final String DELIMITERS = "\\s*[^a-zA-Z]+\\s*";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder allText = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            String value = reader.readLine();
+            while (value != null) {
+                allText.append(value).append(" ");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file!");
```

Comment:
will you have the stack trace of methods that causes this exception and what was the file?